flake.nix

```diff
@@ -0,0 +1,17 @@
+{
+  inputs.ride.url = github:Dyalog/ride;
+  inputs.ride.flake = false;
```

Comment:
Are these inputs ever used?

Reply:
Not yet. It's groundwork for accessing RIDE's source code more conveniently

Reply:
Someone else can build onto that work. I don't think I'll have the time